dashboard/app/models/lesson.rb

```diff
@@ -485,7 +495,7 @@ def summarize_for_lesson_show(user, can_view_teacher_markdown)
       position: relative_position,
       lockable: lockable,
       key: key,
-      displayName: localized_name,
+      displayName: localized_name_for_lesson_show,
```

Comment:
Are you doing this to help with the clean up? Will we remove this once all the HOC editing is done?

Reply:
> Are you doing this to help with the clean up? 

I'm not sure I understand exactly what you're asking. this makes it so that lesson editors get some visual feedback when they change the lesson name, i.e. they will see it on the teacher lesson plan page as opposed to only seeing it on the lesson edit page. 

> Will we remove this once all the HOC editing is done?

yes, we'll remove this once the hoc editing is done. currently I've got that cleanup work tracked here: https://codedotorg.atlassian.net/browse/PLAT-858

Reply:
Sounds good. Was just checking why we were adding this separate method